backend/src/main/java/fk/prof/backend/http/ConfigurableHttpClient.java

```diff
@@ -0,0 +1,161 @@
+package fk.prof.backend.http;
+
+import io.vertx.core.Future;
+import io.vertx.core.Vertx;
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.http.HttpClient;
+import io.vertx.core.http.HttpClientOptions;
+import io.vertx.core.http.HttpClientRequest;
+import io.vertx.core.http.HttpMethod;
+
+public class ConfigurableHttpClient {
```

Comment:
I know, didn't want to give it the name "HttpClient" as that is the name for vertx's http client. Wanted to indicate in some way that is a wrapper over vertx's client. Not a great name though